specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/LinkedService.json

```diff
@@ -4501,6 +4536,38 @@
         "encryptedCredential": {
           "description": "The encrypted credential used for authentication. Credentials are encrypted using the integration runtime credential manager. Type: string (or Expression with resultType string).",
           "type": "object"
+        },
+        "servicePrincipalId": {
```

Comment:
As I know we don't plan to support SPN auth in PostgreSqlV2, please double confirm

Reply:
We are adding entra support, service principal key/cert, system-assigned managed identity/user-assigned managed identity